include/onnxruntime/core/graph/graph.h

```diff
@@ -1380,17 +1396,31 @@ class Graph {
         Graph* parent_graph,
         const Node* parent_node,
         const logging::Logger& logger,
-        bool strict_shape_type_inference);
+        bool strict_shape_type_inference,
+        std::function<void(const InlinedHashSet<std::string>&)> model_functions_remover);
 
   ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE(Graph);
 
+  void RemoveLocalFunctions(const InlinedHashSet<std::string>& retained) {
```

Comment:
```suggestion
  void RemoveModelLocalFunctions(const InlinedHashSet<std::string>& retained) {
```

Reply:
This is a method of the `Model` so I thought that would be redundant.